### PR TITLE
update supported linux platforms

### PR DESCRIPTION
**Pull request by jnweiger**
into master, into 2.4.0 branch?

### CODE REVIEW

**Comment by ogoffart** [comment]
If that's for 2.4, it should be done in the 2.4 branch!

File: doc/installing.rst [hunk 1]
```diff
@@ -30,10 +30,10 @@ System Requirements
 - Windows 7+
 - Mac OS X 10.7+ (**64-bit only**)
 - CentOS 6 & 7 (64-bit only)
-- Debian 7.0 & 8.0 & 9.0
-- Fedora 24 & 25 & 26
-- Ubuntu 16.04 & 16.10 & 17.04
-- openSUSE Leap 42.1 & 42.2 & 42.3
+- Debian 8.0 & 9.0
+- Fedora 25 & 26 & 27
+- Ubuntu 16.04 & 16.10 & 17.10
```
Comment (ogoffart) [inline doc/installing.rst]: so 17.04 is no longer supported while 16.10 still is?  That's a bit strange.

**Comment by guruz** [comment]
@jnweiger ping?

File: doc/installing.rst (hunk 1 above)
Comment (jnweiger) [reply doc/installing.rst]: That seems wrong. Good catch.
The last line should be Ubuntu 16.04 & 17.04 & 17.10 unless @michaelstingl knows better.

**Comment by ogoffart** [comment]
@jnweiger ping?

**Comment by ogoffart** [comment]
(I took the liberty to fixup the line from the github interface)

**Comment by ogoffart** [comment]
Well, just merged.

**Pull request merged**